docs/source/en/guides/cli.md

```diff
@@ -53,6 +53,33 @@ The `--help` option is very convenient for getting more details about a command.
 
 </Tip>
 
+### Alternative Install
+
+#### Using pkgx
+[Pkgx](https://pkgx.sh) is a blazingly fast cross platform package manager that runs anything. It is a single binary that can be installed on any platform. It is a great alternative to pip and conda. You can install huggingface-cli using pkgx as follows:
```

Comment:
```suggestion

[Pkgx](https://pkgx.sh)  is a blazingly fast cross platform package manager that runs anything. You can install huggingface-cli using pkgx as follows:
```

(nit) I would not mention "it is a great alternative to pip and conda" without explaining why it's a great alternative. I think we can keep it simple since we are already referencing to https://pkgx.sh anyway.